improver_tests/cli/test_init.py

```diff
@@ -165,113 +167,94 @@ def test_with_output(self, m):
-        constraints (str or None):
-            Expected name of cube for extraction.
+    These cubes should be the same as the setup cubes.
 
     Returns:
-        iris.cube.Cube:
-            The extracted cube.
-
-    Raises:
-        ValueError:
-            Error to replicate the behaviour of the call of maybe_coerce_with,
-            where loading a cube with an invalid constraint results in a
-            ValueError.
+        iris.cube.CubeList:
+            The CubeList.
     """
-    del func
-    if constraints:
-        cube = cube.extract(constraints)
-    if isinstance(cube, CubeList):
-        (cube,) = cube.copy()
-    if cube is None:
-        raise ValueError
-    return cube
+    return CubeList(
+        [
+            set_up_variable_cube(
+                data=np.zeros((2, 2), dtype=np.float32), name="wind_speed"
+            ),
+            set_up_variable_cube(
+                data=np.zeros((2, 2), dtype=np.float32), name="wind_from_direction"
+            ),
+        ]
+    )
 
 
 class Test_create_constrained_inputcubelist_converter(unittest.TestCase):
-    """Tests the creature constraint_inputcubelist_converter"""
+    """Tests the create constraint_inputcubelist_converter"""
 
     def setUp(self):
         data = np.zeros((2, 2), dtype=np.float32)
         self.wind_speed_cube = set_up_variable_cube(data, name="wind_speed")
         self.wind_dir_cube = set_up_variable_cube(data, name="wind_from_direction")
         self.wind_cubes = CubeList([self.wind_speed_cube, self.wind_dir_cube])
 
-    @patch("improver.cli.maybe_coerce_with", return_value="return")
-    def test_basic(self, m):
-        """Tests that it returns a function which itself returns 2 cubes"""
-        result = create_constrained_inputcubelist_converter(
-            "wind_speed", "wind_from_direction"
+    def test_basic(self):
+        """Tests a basic creation of create_constrained_inputcubelist_converter"""
+        func = create_constrained_inputcubelist_converter(
+            lambda cube: cube.name() in ["wind_speed"],
+            lambda cube: cube.name() in ["wind_from_direction"],
         )
-        result("foo")
-        m.assert_any_call(load_cube, "foo", constraints="wind_speed")
-        m.assert_any_call(load_cube, "foo", constraints="wind_from_direction")
-        self.assertEqual(m.call_count, 2)
-
-    @patch("improver.cli.maybe_coerce_with", return_value="return")
-    def test_list(self, m):
-        """Tests that a list returns a function which itself returns 2 cubes"""
-        result = create_constrained_inputcubelist_converter(
-            ["wind_speed"], ["wind_from_direction"]
+        result = func(self.wind_cubes)
+        self.assertIn(self.wind_speed_cube, result)
+        self.assertIn(self.wind_dir_cube, result)
+        self.assertEqual(2, len(result))
+
+    @patch("improver.cli.maybe_coerce_with", return_value=setup_for_mock())
+    def test_basic_given_str(self, m):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Argument name "m" doesn't conform to snake_case naming style](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5432892)